vendor/src/github.com/docker/swarmkit/api/ca.proto

```diff
@@ -43,6 +43,9 @@ message IssueNodeCertificateRequest {
 	// Token represents a user-provided string that is necessary for new
 	// nodes to join the cluster
 	string token = 3;
+
+	// Availability allows a user to control the current scheduling status of a node.
+	string availability = 4;
```

Comment:
This will be ignored for a renewal. I think that's okay, but we need to make it clear in the comment.

I'm not sure we want to add `availability` to the top level of `IssueNodeCertificateRequest`. Either there should be a nested messages with parameters for the new node object, or we should pass a `NodeSpec` and only honor certain fields from it (because not everything in a `NodeSpec` is under the node's control).

@aluzzardi @stevvooe any thoughts?